### PR TITLE
feat(lidartag): Added support for additional lidars

### PR DESCRIPTION
**Pull request by knzo25**
## PR Type

- New Feature

## Description

Added support for the following lidars:
 - Pandar64
 - AT128
 - OS1 128
 - OS2 64

## Pre-Review Checklist for the PR Author

**PR Author should check the checkboxes below when creating the PR.**

- [ ] Code follows [coding guidelines][coding-guidelines]
- [ ] Assign PR to reviewer

## Checklist for the PR Reviewer

**Reviewers should check the checkboxes below before approval.**

- [ ] Commits are properly organized and messages are according to the guideline
- [ ] Code follows [coding guidelines][coding-guidelines]
- [ ] (Optional) Unit tests have been written for new behavior
- [ ] PR title describes the changes

## Post-Review Checklist for the PR Author

**PR Author should check the checkboxes below before merging.**

- [ ] All open points are addressed and tracked via issues or tickets
- [ ] Write [release notes][release-notes]
- [ ] (When added something to `.repos`) Check if proper access rights are set

## CI Checks

- **vcs import**: Required to pass before the merge.
- **Check spelling**: NOT required to pass before the merge. It is up to the reviewer(s). See [here][spell-check-dict] if you want to add some words to the spell check dictionary.

[coding-guidelines]: https://tier4.atlassian.net/wiki/spaces/AIP/pages/1194394777/T4
[release-notes]: https://tier4.atlassian.net/wiki/spaces/AIP/pages/563774416
[spell-check-dict]: https://github.com/tier4/autoware-spell-check-dict#how-to-contribute

### CODE REVIEW

**Comment by knzo25** [comment]
Confirmed support for OS1-128 !
https://github.com/tier4/CalibrationTools/issues/89

File: launch/lidartag_vader.launch.xml [hunk 1]
```diff
@@ -0,0 +1,68 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (drwnz) [inline launch/lidartag_vader.launch.xml]: I think we don't need this file - we can use at128.

**Review by drwnz — COMMENTED** [review]
Generally LGTM - one question: are the Regents of The University of Michigan copyrights required? If we are now using different sensors with different parameters I'm not sure if they are needed, or if we should add a TIER IV or Autoware Foundation one in addition.

File: launch/lidartag_vader.launch.xml (hunk 1 above)
Comment (knzo25) [reply launch/lidartag_vader.launch.xml]: Addressed in [5d95553](https://github.com/tier4/LiDARTag/pull/15/commits/5d955532a2c6287e6ebc96ebb006bc36ec772650)

**Comment by knzo25** [comment]
> Generally LGTM - one question: are the Regents of The University of Michigan copyrights required? If we are now using different sensors with different parameters I'm not sure if they are needed, or if we should add a TIER IV or Autoware Foundation one in addition.

Just in case, I would prefer to leave both, but so far in the calibration tools we haven't used headers in the launchers.
Should we them in all launchers from now on?

**Comment by drwnz** [comment]
> 

Let's worry about the headers later and merge this for now :)

**Review by drwnz — APPROVED** [review]
LGTM!

**Pull request merged**